eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -898,10 +898,10 @@ object Helpers {
        * @param tx           the remote commitment transaction that has just been published
        * @return a list of transactions (one per output of the commit tx that we can claim)
        */
-      def claimCommitTxOutputs(keyManager: ChannelKeyManager, commitments: Commitments, remoteCommit: RemoteCommit, tx: Transaction, currentBlockHeight: BlockHeight, onChainFeeConf: OnChainFeeConf)(implicit log: LoggingAdapter): RemoteCommitPublished = {
+      def claimCommitTxOutputs(keyManager: ChannelKeyManager, commitments: Commitments, finalScriptPubKey: ByteVector, remoteCommit: RemoteCommit, tx: Transaction, currentBlockHeight: BlockHeight, onChainFeeConf: OnChainFeeConf)(implicit log: LoggingAdapter): RemoteCommitPublished = {
```

Comment:
Nit: I would place the new arg at the end. It makes sense to have the target last, and also I understand that you kept it close to where it was before (in `Commitments`), but the meaning is different now. I think `Commitments` and `RemoteCommit` should be kept together. Same comment applies elsewhere in the file.

```suggestion
      def claimCommitTxOutputs(keyManager: ChannelKeyManager, commitments: Commitments, remoteCommit: RemoteCommit, tx: Transaction, currentBlockHeight: BlockHeight, onChainFeeConf: OnChainFeeConf, finalScriptPubKey: ByteVector)(implicit log: LoggingAdapter): RemoteCommitPublished = {
```

Reply:
done in https://github.com/ACINQ/eclair/pull/2565/commits/044eca1e7390c59990fa78a6086b95de6f895b8b